backend/corpora/lambdas/api/v1/curation/collections/collection_uuid/assets.py

```diff
@@ -0,0 +1,56 @@
+from flask import g, make_response, jsonify
+
+from backend.corpora.api_server.db import dbconnect
+from backend.corpora.common.entities import Dataset, DatasetAsset
+from backend.corpora.common.utils.http_exceptions import InvalidParametersHTTPException, NotFoundHTTPException
+
+
+@dbconnect
+def get(collection_uuid: str, curator_tag: str = None, dataset_uuid=None):
+    db_session = g.db_session
+    if dataset_uuid:
+        dataset = Dataset.get(db_session, dataset_uuid)
```

Comment:
`collection_uuid` is not used here to fetch the Dataset; as a consequence, if `dataset_uuid` is used, then `collection_uuid` is not required to be correct in order to fetch the Dataset in question.

I think this operation should always _fail_ if the id for the Collection is wrong.

Reply:
I don't think this represents a security issue as `dataset_uuid` can be considered a 'capability identifier', i.e. if you have it, then you should have the right to view this.

But we may wish to express that to curators as well.

Reply:
Added check for collection